src/test/java/org/mule/extension/file/integration/FileWriteTestCase.java

```diff
@@ -32,10 +32,16 @@
 import org.mule.runtime.core.api.event.CoreEvent;
 
 import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.lang.reflect.Method;
 import java.util.Arrays;
+import java.util.List;
 
-import io.qameta.allure.Feature;
+import org.junit.Ignore;
```

Comment:
Is this being used?

Reply:
Yes, by another test not related to this PR. It was already imported, I only switched the import order.